ffig/filters/capi_filter.py

```diff
@@ -51,7 +51,7 @@ def to_go(t):
         if t.pointee.kind == TypeKind.CHAR_S:
             return 'string'
         if t.pointee.kind == TypeKind.RECORD:
-            return 'unsafe.Pointer'
+            return t.pointee.name.replace('const', '')
```

Comment:
Hmm, that's true. I'll change it to add the space. of course, then it would still match `Foo_const`, which I suppose is less likely to occur, but still a valid identifier.